include/pistache/stream.h

```diff
@@ -159,12 +159,12 @@ class DynamicStreamBuf : public StreamBuf<char> {
   }
 
   RawBuffer buffer() const {
-    return RawBuffer((const char *)data_.data(), pptr() - &data_[0]);
+    return RawBuffer((const char*) data_.data(), pptr() - data_.data());
```

Comment:
Can you please replace the C style type cast with `static_cast`?

Reply:
actually, let me see if the cast can be removed altogether.